server/claim-review/schemas/claim-review.schema.ts

```diff
@@ -13,7 +13,7 @@ export class ClaimReview {
     @Prop({
         type: mongoose.Types.ObjectId,
         required: true,
-        ref: "ClaimRevision",
+        ref: "Claim",
```

Comment:
Shouldn't the ClaimReview reference the Sentences now?

Reply:
The claim Review schema has the report field with the id, and the report schema has the sentence_hash field. Is a claim Review schema reference sentence necessary?

Reply:
@pepermao you're right. At least not at the moment, maybe when we start looking for performance gains we can revisit it.